apps/service/src/app/globals.css

```
@@ -35,3 +35,22 @@
   --animate-accordion-down: accordion-down 0.2s ease-out;
   --animate-accordion-up: accordion-up 0.2s ease-in;
 }
+
+/* 모바일 꾹 누르기 메뉴 및 텍스트 선택 방지 */
+* {
+  -webkit-touch-callout: none; /* iOS Safari: 링크/이미지 꾹 누를 때 메뉴 방지 */
+  -webkit-user-select: none;    /* Safari */
+  -khtml-user-select: none;     /* Konqueror HTML */
```

Biome (2.4.7)

[error] Unknown property is not allowed.

(lint/correctness/noUnknownProperty)